impl/client/src/main/javascript/web/prompting/api/UiAPI.js

```diff
@@ -87,5 +88,19 @@ define([], function() {
     this.setBlockUiOptions = function(options) {
       api.operation._getPromptPanel().setBlockUiOptions(options);
     };
+
+    /**
+     * Enables or disables a submit button in a submit panel. Checks if it's necessary change "disabled" attribute of
+     * the button and applies it to a submit button component in the dashboard.
+     *
+     * @name UiAPI#setDisabledSubmitButton
+     * @method
+     * @param {Boolean} disabled The flag to enable (use `false` flag) or disable (use `true` flag) the submit button.
```

Comment:
Could also be described as (same for `promptPanel.setDisabledSubmitButton`):
*When `true` enables the submit button, when `false` disables it.*